src/main/org/codehaus/groovy/classgen/Verifier.java

```diff
@@ -384,6 +387,7 @@ public void visit(MethodVisitor mv) {
     protected void addGroovyObjectInterfaceAndMethods(ClassNode node, final String classInternalName) {
         if (!node.isDerivedFromGroovyObject()) node.addInterface(ClassHelper.make(GroovyObject.class));
         FieldNode metaClassField = getMetaClassField(node);
+        AnnotationNode generatedAnnotation = new AnnotationNode(ClassHelper.make(GENERATED_ANNOTATION));
 
```

Comment:
I needed a guard on adding the annotation to keep the build happy:
```
boolean shouldAnnotate = classNode.getModule().getContext() != null;
```

Reply:
2 small fixes added 😄